streams.json

```diff
@@ -257,6 +257,55 @@
           "codebase": "Branch_JGroups_3_2"
         },
         {
+          "component_name": "JBoss JSF API",
+          "repository_url": "https://github.com/jboss/jboss-jsf-api_spec/",
+          "codebase": "2.1.28"
+        },
+        {
+          "component_name": "Marshalling",
+          "repository_url": "https://github.com/jboss-remoting/jboss-marshalling",
+          "codebase": "1.4"
+        },
+	    {
+          "component_name": "Netty",
+          "repository_url": "https://github.com/netty/netty",
+          "codebase": "3.6"
+        },
+	    {
+          "component_name": "Metadata",
```

Comment:
If we avoid minor things like this, I'm sure the final will be easier to read.  Better to be strict at the beginning, so we don't end up with an unreadable mess later :)